content/influxdb/v2.1/security/tokens/create-token.md

```diff
@@ -15,26 +15,90 @@ Create API tokens using the InfluxDB user interface (UI), the `influx`
 command line interface (CLI), or the InfluxDB API.
 
 {{% note %}}
-Tokens are visible to the user who created the token.
-{{% oss-only %}}Users who own a token with Operator permissions also have access to all tokens.{{% /oss-only %}}
+
+{{% oss-only %}}Tokens are visible to the user who created the token. Users who own a token with Operator permissions also have access to all tokens.
 Tokens stop working when the user who created the token is deleted.
+
 **We recommend creating a generic user to create and manage tokens for writing data.**
+{{% /oss-only %}}
+
+{{% cloud-only %}}
+To follow best practices for secure API token generation and retrieval, InfluxDB Cloud enforces access restrictions on API tokens.
+  - Tokens are visible to the user who created the token.
+  - InfluxDB Cloud UI only allows access to the API token value immediately after the token is created.
+  - You can't change access (**read/write**) permissions for an API token after it's created.
+  - Tokens stop working when the user who created the token is deleted.
+
+We recommend the following for managing your tokens:
+- Create a generic user to create and manage tokens for writing data.
+- Store your tokens in a secure password vault for future access.
+{{% /cloud-only %}}
 {{% /note %}}
 
+- [Create a token in the InfluxDB UI](#create-a-token-in-the-influxdb-ui)
+- [Create a token using the influx CLI](#create-a-token-using-the-influx-cli)
+- [Create a token using the InfluxDB API](#create-a-token-using-the-influxdb-api)
+
 ## Create a token in the InfluxDB UI
 
+{{% oss-only %}}
+
 1. In the navigation menu on the left, select **Data (Load Data)** > **Tokens**.
```

Comment:
```suggestion
1. In the navigation menu on the left, select **Load Data** > **API Tokens**.
```

Reply:
When OSS nav menu is collapsed (which I think is the default), it says "Data".